src/components/home/SideBarNavLink.tsx

```diff
@@ -38,7 +38,7 @@ const SideBarNavLink: FunctionComponent<SideBarNavLinkProps> = ({
   onClick,
 }) => {
   const resolved = useResolvedPath(to || "");
-  const match = useMatch(resolved.pathname);
```

Comment:
Updated this match so that any url path with example "/library/*" makes the "library" section of the sidenav as selected